src/ergw_aaa_radius.erl

```diff
@@ -737,8 +747,10 @@ radius_accounting_opts() ->
 		       ?Acct_Status_Type,
 		       ?Event_Timestamp]).
 
-send_request(Req, Session, #{server := NAS} = Opts) ->
+send_request(Req, Session, #{server := NAS, retries := Retries,
+			     timeout := Timeout} = Opts) when is_tuple(NAS)->
     Id = maps:get('NAS-Identifier', Session, <<"NAS">>),
     RadiusClientOpts = [{client_name, maps:get(client_name, Opts, Id)},
 			{server_name, maps:get(server_name, Opts, Id)}],
-    eradius_client:send_request(NAS, Req, RadiusClientOpts).
+    eradius_client:send_request(NAS, Req, RadiusClientOpts ++ 
```

Comment:
why `++`? You could just initialize `RadiusClientOpts` the all options.

Reply:
implemented